swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -332,6 +401,143 @@ public boolean hasSelection() {
 		Object item = getSelectedItem();
 		return item != null && item != nullItem;
 	}
+	
+	/**
+	 * {@inheritDoc }
+	 * Deal with edge cases for combo editor interaction with GlazedLists and lack of support
+	 * for STRICT/CONTAINS in GlazedLists.
+	 */
+	// **** GL STRICT/CONTAINS ****
+	//
+	// IT IS POSSIBLE THAT NO OVERRIDE OF setSelectedItem() WILL BE NEEDED IF GLAZEDLISTS FULLY SUPPORTS
+	// STRICT/CONTAINS
+	@Override
+	public void setSelectedItem(final Object _value) {
+		// 2020-12-30_BP: setSelectedItem outcomes:
+		//
+		// See https://docs.oracle.com/javase/8/docs/api/javax/swing/JComboBox.html#setSelectedItem-java.lang.Object-
+		//
+		//  1. We get a null:
+		//		A. A direct/indirect non-UI call was made to setSelectedItem(null).
+		//         This could be a value from the rowset listener, a programmatically set default, etc.
+		//		   In this case the editorComponent should NOT have the focus.
+		//         -> call super.setSelectedItem(null) and continue
+		//      B. GlazedList bug where for the first character entered, the START_WITH matching is performed over the CONTAINS matching. In this case,
+		//         (getModel().getSize() > 0)
+		//         -> call super.setSelectedItem(getItemAt(0)) and continue
+		//      C. The user's entry into the editor results in zero matches. In this case ((getModel().getSize()==0) && (hasItems()==true))
+		//         -> set actionListenerNoUpdate to true so the Action Listener does not try to update the rowset
+		//         -> warn user, (attempt to) revert the editor string, and continue (simulated STRICT)
+	    //      D. SOMETHING ELSE???
+		//         -> set actionListenerNoUpdate to true so the Action Listener does not try to update the rowset
+		//         -> warn user and continue
+		//         
+		//  2. We get a valid SSListItem including nullItem if getAllowNull()==true. This could be from the UI or from a 
+		//     direct/indirect non UI call to setSelectedItem().
+		//	   -> call super.setSelectedItem(_value) and continue
+		//
+		//  3. We get a String or something else.
+		//     -> set actionListenerNoUpdate to true so the Action Listener does not try to update the rowset
+		//     -> warn user and return
+	
+		// INITIALIZATION
+		glGlitchItem = null; // WORKAROUND FOR GL 'CONTAINS' GLITCH
+		actionListenerNoUpdate = false; // TRUE WHEN THE USER HAS ENTERED SOMETHING RESULTING IN NO MATCHES
+
+		// GET CURRENT EDITOR TEXT IF AVAILABLE
+		final String currentEditorText;
+		if (getEditor().getItem() == null) {
```

Comment:
I think I see it now. If getEditor() returned null then `getEditor().getItem() == null` above should throw an NPE. Since it hasn't in testing, I guess we're safe.

I removed and committed.